<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyBrowser</class>
 <widget class="QWidget" name="MyBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_proxy"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_use_proxy">
       <property name="text">
        <string>use</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_user_agent"/>
     </item>
     <item>
      <widget class="QLineEdit" name="le_address"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_run">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebEngineView" name="webEngineView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
